fffw/encoding/outputs.py

```diff
@@ -93,6 +93,17 @@ def check_buffering(self) -> Optional[List[str]]:
             prev = scene
         return meta.streams
 
+    @property
+    def meta(self) -> Optional[Meta]:
+        metadata = self.get_meta_data(self)
+        if metadata is None:
+            return None
+        return self.transform(metadata)
+
+    def transform(self, metadata: Meta) -> Meta:
+        """ Apply filter changes to codec metadata."""
```

Comment:
"Apply codec changes to stream metadata."

Reply:
fixed